packages/ceramic-doctype-tile/src/tile-doctype.ts

```diff
@@ -114,6 +126,20 @@ export class TileDoctype extends Doctype {
         return TileDoctype._signDagJWS(record, did, doctype.owners[0])
     }
 
+    /**
+     * Calculates anchor nonce
+     */
+    private static async _calculateNonce(doctype: Doctype): Promise<number> {
+        if (doctype.context.ipfs == null) {
+            return 0 // return nonce 0 if IPFS is not available
+        }
+
+        // If there hasn't been any update prior to this we should not set the nonce
+        if (!doctype.state.next) return
+        // Get the current nonce and increment it by one
+        return (doctype.state.next.metadata?.nonce || 0)++
```

Comment:
```suggestion
        return ++(doctype.state.next.metadata?.nonce || 0)
```

Reply:
With this change you proposed the `log`s in the tests are the same. I'll take a look of how to implement the optimization you proposed (squashing the updates).

Reply:
That's great, ty!